DebugEngine/MagoNatDE/MagoNatDE.vcxproj

```diff
@@ -41,14 +41,14 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
-    <ConfigurationType>DynamicLibrary</ConfigurationType>

```

Comment:
Changing the Debug/Release configurations should no longer be necessary. Just build with the "Debug/Release StaticDE" solution configuration. (It should still be possible to build MagoNatDE.dll as the debugger engine for VS).